airflow/www/templates/airflow/dag.html

```diff
@@ -91,11 +91,15 @@ <h4 class="pull-right" style="user-select: none;-moz-user-select: auto;">
       <a class="label label-default" href="{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}">
         Schedule: {{ dag.schedule_interval }}
       </a>
+      {% if dag.timetable.interval_description is not none %}
+      <span id="schedule-description-{{ dag.safe_dag_id }}" class="material-icons text-muted js-tooltip" aria-hidden="true" data-original-title="Runs: {{ dag.timetable.interval_description|string }}">info</span>
+      {% endif %}
```

Comment:
`dag` here is not always a `DAG` object, but may be a `DagModel`. So this will not work. You need to introduce a common interface across `DAG` and `DagModel` instead of accessing `timetable` directly. See `DAG.bulk_write_to_db()` for how `DAG` attributes are written to corresponding `DagModel` attributes.

Reply:
sure, let me check,
but as far as I know we always get DagModel object in dag.html,
what are the cases where we will have Dag object.